ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/views/OntologiesViewImpl.java

```diff
@@ -276,18 +300,14 @@ public void onSelection(SelectionEvent<Ontology> event) {
         return ontologySimpleComboBox;
     }
 
-    void updateButtonStatus() {
+    public void updateButtonStatus() {
         publishButton.setEnabled(selectedOntology != null && selectedOntology != activeOntology);
         saveHierarchy.setEnabled(selectedOntology != null);
+        deleteButton.setEnabled(selectedOntology != null && selectedOntology != activeOntology);
+        deleteHierarchy.setEnabled(selectedOntology != null && treeStore.getRootItems().size() > 0);
```

Comment:
True.  I guess my worry was that we're not allowing the admins to delete the active ontology outright, but if you accidentally deleted all the hierarchies in the active ontology, that's sort of the same?